lesson_25/jimoye244a/src/pages/CreateProgram/CreateProgram.scss

```diff
@@ -0,0 +1,91 @@
+/* App.css */
+
+* {
```

Comment:
Don't need this and the body style here. Remember that this component gets "slotted in" to the App.tsx layout that already includes the App.css styles and common layout shared by routes that leverage it (see main.tsx).